meilisearch-core/src/update/documents_addition.rs

```diff
@@ -289,6 +245,11 @@ pub fn reindex_all_documents(writer: &mut heed::RwTxn<MainT>, index: &store::Ind
     let mut indexer = RawIndexer::new(stop_words);
     let mut ram_store = HashMap::new();
 
+    if let Some(ref attributes_for_facetting) = index.main.attributes_for_faceting(writer)? {
+        let facet_map = facets::facet_map_from_docids(writer, &index, &documents_ids_to_reindex, &attributes_for_facetting)?;
+        index.facets.add(writer, facet_map)?;
+    }
+    // ^-- merge? --v
```

Comment:
What this comment means again?

Reply:
what facet map does could be done inside the loop under the comment, and save us a few db reads. this would be done at the price of less readability, and would make it harder to move this logic out if we wanted it. This comment sole purpose is to tell us that this optimization is possible, is we want it, as it is not trivial.